### DISCUSSION TITLE
Issues with white pages since v4 migration.

### DISCUSSION THREAD

**Discussion by benrobson** [fastify/fastify#5080] [Q&A]
Hey,
I am having issues migrating from v3 to v4, I updated all of the packages but it's now just throwing up white pages, I've added my code below:
Error: https://hastebin.com/share/vamojifijo.swift
Main Route: https://hastebin.com/share/edusukihec.javascript
API Stats Route: https://hastebin.com/share/afobovutav.csharp

**Comment by mcollina** [comment]
Thanks for reporting!

Can you provide steps to reproduce? We often need a [reproducible example](https://stackoverflow.com/help/minimal-reproducible-example), e.g. some code that allows someone else to recreate your problem by just copying and pasting it. If it involves more than a couple of different file, create a new repository on GitHub and add a link to that.

**Reply by benrobson** [reply]
Could I invite you to the repo instead of doing a reproducible example?

**Reply by Eomm** [reply]
That would be consulting

**Comment by benrobson** [comment]
Fixed it myself.
